src/app/fyle/split-expense/split-expense.page.ts

```
@@ -117,8 +127,8 @@
 
   unspecifiedCategory: OrgCategory = null;
 
-  splitExpenseHeader: string;
-
+  isReviewModalOpen = false;
+  
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)